knitr/planetary_motion/planetary_motion.rmd

```diff
@@ -0,0 +1,735 @@
+---
+
+# Introduction
+
+As developers of statistical softwares^[The authors are notably members of the Stan development team, see [mc-stan.org](https://mc-stan.org/).], we realize that we cannot fully automate modeling.
+Practitioners need to take bespoke steps to fit, evaluate, and improve their models.
+At the same time, the more modeling we do, the better prepared we usually are for the next project we undertake.
+It's not uncommon to apply hard-learned lessons from a past project to a new problem.
+Articles on _modeling workflows_ aim to formalize this experience, building and deconstructing heuristics, developing methods, and motivating new theory [e.g. @Blei:2014; @Gabry:2019; @Betancourt:2020; @Gelman:2020].
+We believe this effort must be complemented by case studies, such as this one, which illustrate in great details the concepts that arise in workflows, provide usable code, and also raise new questions.
+The specific concepts we encounter in this article include why our inference fails way more slowly than it succeeds, why the behavior of an ODE changes wildely across the parameter space, and what to do when chains drift into "unreasonable" modes of the posterior distribution.
+
+Given the recorded position of a planet over time, we want to estimate the physical properties of a star-planet system.
+This includes position, momentum, and gravitational interaction.
+We fit the model with Stan [@Carpenter:2017], using Hamiltonian Monte Carlo (HMC), a gradient-based Markov chain Monte Carlo (MCMC) algorithm;
+for a thorough introduction to HMC, we recommend the article by @Betancourt:2018.
+Initially, we meant the planetary motion problem to be a simple textbook example for ODE-based models; but it turns out many interesting challenges arise when we do a Bayesian analysis on this model.
+We discuss how to diagnose and fix these issues.
+The techniques we deploy involve a combination of mathematical considerations that are specific to the system at hand and statistical principles we can generalize.
+This case study examines the model in the controlled setting of simulated data.
+
+We begin by constructing our model, using elementary notions of classical mechanics.
+After failing to fit this complete model, we develop a simpler model and build our way back to the original model.
+This step-by-step expansion allows us to isolate issues that make Bayesian inference challenging.
+Classic tools we use involve: running multiple chains, convergence diagnostics, and simulating data from the fitted model.
+The latter can be done in several ways and an agile use of plots proves extremely helpful.
```

Comment:
I still don't like latter. It's hard to understand. "The latter can be done in several ways"

1. What can be done in several ways
2. What are those several ways

Tough sentence to dig into, cuz it's so vague. I don't think it's bad to just leave it but since there was no thumbs up thumbs down or anything so I'm following up on it.

Also switch "convergence diagnostics" to "checking convergence diagnostics". Faulty parallelism otherwise